response_operations_ui/controllers/case_controller.py

```diff
@@ -107,3 +107,23 @@ def is_allowed_status(status):
 def get_case_group_status_by_collection_exercise(case_groups, collection_exercise_id):
     return next((case_group['caseGroupStatus'] for case_group in case_groups
                  if case_group['collectionExerciseId'] == collection_exercise_id), None)
+
+
+def get_cases_by_sample_unit_id(sample_unit_ids):
+    logger.debug('Retrieving cases for sample unit IDs', sample_unit_ids=sample_unit_ids)
+    url = f'{app.config["CASE_URL"]}/cases/sampleunitids'
```

Comment:
Getting a 500 from the case service when testing locally.

Example: `http://localhost:8171/cases/sampleunitids?sampleUnitId=a7656f07-07ac-4484-b992-a81fd51451eb`

"Failed to convert value of type 'java.lang.String' to required type 'java.util.UUID'; nested exception is java.lang.IllegalArgumentException: Invalid UUID string: sampleunitids"

Reply:
Sorted now ^ Local issue with out of date docker images.